penpotdesktop.desktop

```
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Penpot Desktop
+Comment=Design Freedom for Teams
+Exec=com.korbsstudio.penpot-desktop
```

This doesn't match the `command` in the manifest
```suggestion
Exec=run.sh
```